pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -458,6 +470,9 @@ message BaseCommand {
 		REACHED_END_OF_TOPIC = 27;
 
 		SEEK = 28;
+
+		GET_TOPICS_OF_NAMESPACE 			= 29;
```

Comment:
If I remembered correctly, we are bumping the protocol version in one of your other pull requests, if this change is going to be merged with that pull request in same release, I would suggest update the comment of that protocol version with these two new commands.

Otherwise, we have to bump the protocol version.

Reply:
Yes. there was #1066 to `get the last message id`, which is marked as 1.22.